kratos/processes/find_intersected_geometrical_objects_with_obb_process.h

```diff
@@ -202,40 +202,40 @@ class KRATOS_API(KRATOS_CORE) FindIntersectedGeometricalObjectsWithOBBProcess
      * @param rFirstGeometry The first geometry
      * @param rSecondGeometry The second geometry
      */
-    bool HasIntersection2D(
+    virtual bool HasIntersection2D(
```

Comment:
Couldn't we use the same mechanism as in the standard find intersected geometrical object process in the OBB one?

Reply:
I agree, but maybe in another PR

Reply:
not sure how the OBB intersections work

Reply:
I think that the unique change is the way you construct the bounding box of the tree, so I guess intersection would be the same.